docs: clarify description of Peek-A-Boo example's lifecycle hooks (#4… (#40897)
angular/angular: aio/content/guide/lifecycle-hooks.md

### Sequence and frequency of all lifecycle events

> To show how Angular calls the hooks in the expected order, the `PeekABooComponent` demonstrates all of the hooks in one example.

Reviewer: Why this change? The component does demonstrate all the hooks, no?

Author: Only because it sounds repetitious, IMO; "the peekaboo component demos...in one component".

  <img src="generated/images/guide/lifecycle-hooks/peek-a-boo.png" alt="Peek-a-boo">
</div>

The sequence of log messages beginning at #2 displays the prescribed hook calling order.

Note that `DoCheck` as well as the subsequent `AfterContentChecked` and `AfterViewChecked` appear more than once.
This is because even though the `DoCheck` hook exists in the child component,
it is called in every change detection cycle _anywhere on the page_.

<div class="alert is-helpful">

  Notice that the log confirms that input properties (the `name` property in this case) have no assigned values at construction.
  The input properties are available to the `onInit()` method for further initialization.

> k

Reviewer 2: revert???

Author: Just removed in latest push...

  <img src="generated/images/guide/lifecycle-hooks/peek-a-boo.png" alt="Peek-a-boo">
</div>

The sequence of log messages beginning at #2 displays the prescribed hook calling order.

> Note that `DoCheck` as well as the subsequent `AfterContentChecked` and `AfterViewInit` appear more than once.

Reviewer 3: I only see one instance of 'AfterViewInit' in graphic.

Author: Thanks for catching that.
Should have been AfterContentChecked and **AfterViewChecked**
Change made in fixup push.
Thanks!

> This is due to the fact that even though the `DoCheck` hook exists in the child component,

Reviewer 3: ```suggestion
This is because even though the `DoCheck` hook exists in the child component,
```
Tightened text.

Author: Agreed. Change made in latest push. Thanks!
